### PR TITLE
created: creative card

### PR DESCRIPTION
**Pull request by mmusk**
Created a Card with hover effects using HTML/CSS

### CODE REVIEW

File: tasks-by-users/mmusk/Creative Card/style.css [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+
+body{
+    display: flex;
+    justify-content: center;
+    align-content: center;
+
+    min-height: 100vh;
+
+    background-color: #131313;
+}
```
Comment (coder-mezab) [inline tasks-by-users/mmusk/Creative Card/style.css:18]: wrap this in one big container its not a good practice to give body properties @mmusk

File: tasks-by-users/mmusk/Creative Card/style.css [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+
+body{
+    display: flex;
+    justify-content: center;
+    align-content: center;
+
+    min-height: 100vh;
+
+    background-color: #131313;
```
Comment (coder-mezab) [inline tasks-by-users/mmusk/Creative Card/style.css:17]:
```suggestion
    background: #131313;
```

File: tasks-by-users/mmusk/Creative Card/style.css [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+    position: relative;
+}
+
+.container .card{  
+    position: relative;
+
+    width: 320px;
+    height: 450px;
+
+    background-color: #232323;
```
Comment (coder-mezab) [inline tasks-by-users/mmusk/Creative Card/style.css:30]:
```suggestion
    background: #232323;
```

File: tasks-by-users/mmusk/Creative Card/style.css [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+
+    transition: 0.5s;
+
+    opacity: 0;
+
+    visibility: hidden;
+}
+
+.container .card:hover .contentbx .size{
+    opacity: 1;
```
Comment (coder-mezab) [inline tasks-by-users/mmusk/Creative Card/style.css:159]: i think this is not needed it will have default  `opacity: 1 `

**Review by coder-mezab — CHANGES_REQUESTED** [review]

File: tasks-by-users/mmusk/Creative Card/style.css (hunk 1 above)
Comment (mmusk) [reply tasks-by-users/mmusk/Creative Card/style.css]: fixed @coder-mezab

**Review by anask601 — APPROVED** [review]

File: tasks-by-users/mmusk/Creative Card/style.css (hunk 4 above)
Comment (mmusk) [reply tasks-by-users/mmusk/Creative Card/style.css:157]: It is needed or else the size option gets disappered @coder-mezab

**Comment by mmusk** [comment]
> requsted some minor changes once done mention me here i will approve @mmusk

done with the changes please check @coder-mezab

**Review by coder-mezab — APPROVED** [review]

**Review by AbbasShaikh0304 — APPROVED** [review]

**Pull request merged**